deployment&configuration/oms/additional-resources/Webtools.md

```diff
@@ -0,0 +1,130 @@
+# Web Tools
+
+Web Tools serve as a valuable resource for both the backend and development teams, offering a comprehensive suite of functionalities for data management, log viewing, data import/export, job execution, and more within an Order Management System (OMS) instance. 
+
+Accessing Web Tools is straightforward. Users can conveniently navigate to the following URL in their web browsers: https://_**user-instance**_.hotwax.io/webtools. For example, for demo-oms, the corresponding URL would be https://demo-oms.hotwax.io/webtools.
+
+Upon reaching the Web Tools portal, users are prompted to log in using their credentials.
+
+___
+
+## Entity Engine
+
+The `Entity Engine` feature within the HotWax Commerce Web Tools plays a crucial role in facilitating the viewing and management of data. It is designed to enhance the user experience by providing a platform for handling entities within the system. This feature is particularly relevant for backend and development teams, allowing them to manage data efficiently.
+
+Entity Engine is accessible through the `Entity Engine` button on the second row of tabs on the main web tools page. Alternatively, users can also find a list of `Entity Engine Tools` directly on the web tools main page.
+
+* Click on the `Entity Engine` button to be redirected to the `Entity Data Maintenance` page. 
+* Use filters such as `Group Name` and `Entity Name` to find specific records.
+* An alphabetical list of all entities is also displayed on this page.
+* Click on the name of a specific entity to view its dataset.
+* Explore functions and filters to search data within the entity.
+* For a comprehensive view of all records of an entity, click the `Search` button.
+* Utilize the `View Relations` option to explore relationships with other entities. For example, if we consider the entity ‘Facility’, we can see that it is related to entities ‘FacilityGroup’, ‘ProductStore’, ‘Party’ and so on. 
+
+
+### Entity SQL Processor
+
+The Entity SQL Processor in Web Tools, serves as a valuable component for interpreting and executing SQL commands. This tool is designed to enhance the efficiency of data viewing and management by providing users with the capability to execute SQL queries in the system. 
+
```

Comment:
@sanjanashrivastava Add a warning here that always use the `Select` query first and then other queries. Also add a warning to refrain from using Delete queries.